File: scratch/inspect_excel.py
import pandas as pd

try:
    df = pd.read_excel('Catalogo_Detallado_Eleodoro.csv final.xlsx')
    print("Columns:")
    print(df.columns.tolist())
    print("\nFirst 3 rows:")
    print(df.head(3).to_string())
    print("\nData shape:", df.shape)
except Exception as e:
    print("Error reading Excel with pandas:", e)

File: scratch/inspect_excel.js
const xlsx = require('xlsx');

try {
    const workbook = xlsx.readFile('Catalogo_Detallado_Eleodoro.csv final.xlsx');
    console.log("Sheet names:", workbook.SheetNames);
    const sheetName = workbook.SheetNames[0];
    const sheet = workbook.Sheets[sheetName];
    const data = xlsx.utils.sheet_to_json(sheet);
    console.log("Number of rows:", data.length);
    if (data.length > 0) {
        console.log("Columns:", Object.keys(data[0]));
        console.log("First row:", JSON.stringify(data[0], null, 2));
        console.log("Second row:", JSON.stringify(data[1], null, 2));
    }
} catch (e) {
    console.error("Error:", e.message);
}
